grammars/jsdoc.cson

```diff
@@ -411,6 +435,120 @@
         ]
       }
     ]
+  # Default @param value, highlighted as JavaScript
+  'default-value':
+    'patterns': [
+      # Double-quoted string
+      {
+        'begin': '(")'
```

Comment:
Would you prefer I simply embed `source.js` in the captured match, then? Unless you want *no* string highlighting whatsoever...